src/Microsoft.IdentityModel.Tokens/TokenHandler.cs

```diff
@@ -75,6 +91,21 @@ public int TokenLifetimeInMinutes
         /// <returns>A <see cref="TokenValidationResult"/></returns>
         public virtual Task<TokenValidationResult> ValidateTokenAsync(SecurityToken token, TokenValidationParameters validationParameters) => throw new NotImplementedException();
 
+        /// <summary>
+        /// Evaluates whether a token can be converted into an instance of <see cref="SecurityToken"/>.
+        /// </summary>
+        /// <param name="token">The string to be evaluated.</param>
+        /// <returns>A <see cref="bool"/>.</returns>
+        public virtual bool CanReadToken(string token) => throw new NotImplementedException();
+
+        /// <summary>
+        /// Creates a token as described by the provided <see cref="SecurityTokenDescriptor"/>.
+        /// </summary>
+        /// <param name="tokenDescriptor">The <see cref="SecurityTokenDescriptor"/> used to create the token.</param>
+        /// <returns>A <see cref="string"/>.</returns>
+        /// <exception cref="NotImplementedException"></exception>
+        public virtual string CreateToken(SecurityTokenDescriptor tokenDescriptor) => throw new NotImplementedException();
```

Comment:
I believe this method already exists on the JsonWebTokenHandler and/or JwtSecurityTokenHandler you'll need to override them as this is causing test to fail.

Reply:
We can remove the method from SecurityTokenHander.

Reply:
We should also wait until we create the 7.x branch, then we can remove the '=> throw new NotImplementedException()" as @keegan-caruso suggested.